noxfile.py

```diff
@@ -0,0 +1,61 @@
+import nox
+import os
+import sys
+import shutil
+
+nox.options.reuse_existing_virtualenvs = True
+
+@nox.session(name="themes")
+def run_themes(session):
+    themes_dir = "themes"
+    if not os.path.exists(themes_dir) or not os.listdir(themes_dir):
+        session.run("git",
+            "submodule",
+            "update",
+            "--init",
+            "--recursive",
+            external=True,
+        )
+    else:
+        print("'themes/' already exists in the project's root.")
+
+
+@nox.session(name="html")
+def build_html(session):
+    # Ideally hugo must be pre-installed
+    if sys.platform=="linux":
+        session.run("apt", "install", "hugo", external=True)
+    session.run("hugo")
```

Comment:
```suggestion
```

I would instead suggest running `hugo --version` as a command here to check if it is installed and present on the PATH. The issue with this is that it doesn't work on all flavours of Linux where we could have different package managers such as `yum`, `dnf`, etc. We can't and don't need to support all of them, so we can just run `session.error("The extended version of Hugo is not installed")` if the command fails

Reply:
> I would instead suggest running `hugo version` as a command here to check if it is installed and present on the PATH.

According to my observation only `hugo` should be enough as both returns the same logs.